airbyte-integrations/connectors/source-salesforce/unit_tests/conftest.py

```diff
@@ -22,14 +23,14 @@ def time_sleep_mock(mocker):
 
 @pytest.fixture(scope="module")
 def bulk_catalog():
-    with open("unit_tests/bulk_catalog.json") as f:
+    with (pathlib.Path(__file__).parent / "bulk_catalog.json").open() as f:
```

Comment:
This has nothing to do with the change described but PyCharm was creating the default current working directory to be `source-salesforce` and not `source-salesforce/unit_tests`. Hence, I made the test agnostic of the current working directory